tests/unit/RoutesTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace PagarMe\Test;
+
+use PagarMe\Routes;
+use PHPUnit\Framework\TestCase;
+
+class RoutesTest extends TestCase
```

Comment:
Are not you testing the programming language instead of the SDK features?

Reply:
@murilohns entendi seu ponto e entendi os motivos do @devdrops ter escrito testes pra isso.

Meus dois centávos é que esses testes são necessários dada a decisão de design que tomamos na hora da implementação. Acredito que a dinamicidade implementada em `PagarMe\Routes` nos trouxe até aqui.

Daí pergunto: essa dinamicidade faz sentido dado que vamos ter que "testar" todas as propriedades da classe pra garantir que elas são `callables`?

_Filosofei mesmo kkkk_

Reply:
Hi @murilohns!

I totally understand your question, because that's exactly what I thought when writing this test (`geez, WTH am I doing?`). However, the implicit necessity of `RoutesTest` is to reinforce that we're developing each route as it should be - you see, between the objects, there are no contracts that explicitly requires that every route should be invoked/callable, neither that we're delivering all the necessary routes for each endpoint. So this is the idea that was in my mind when I created this test: a guarantee that the `Routes` class is behaving as expected.